businessRules/ideas.js

```diff
@@ -20,33 +27,21 @@ function sortedByDonationSumIdeas() {
 }
 
 function createIdeas(header, description, askedSum, userId) {
-    if (!header) {
-        throw new Error('Error. Please, check your header.');
-    }
+    checkIfHeaderProvided(header);
```

Comment:
A clearer function name would be: `isHeaderValid`

Reply:
That situation, when you're getting way too creative and informative while thinking of a new name for the variable.

Reply:
Commited